documentation/api/command/v1/commands.markdown

```diff
@@ -22,25 +24,25 @@ which have the following JSON wire format:
 
 `command` is a string identifying the command.
 
```

Comment:
While I like the new way of line wrapping (M-q?), this makes it hard to see what changed and will break `git blame` down the road.